orchagent/routeorch.cpp

```diff
@@ -18,13 +19,10 @@ extern sai_switch_api_t*            sai_switch_api;
 extern PortsOrch *gPortsOrch;
 extern CrmOrch *gCrmOrch;
 extern Directory<Orch*> gDirectory;
+extern NhgOrch *gNhgOrch;
 
 extern size_t gMaxBulkSize;
 
-/* Default maximum number of next hop groups */
-#define DEFAULT_NUMBER_OF_ECMP_GROUPS   128
-#define DEFAULT_MAX_ECMP_GROUP_SIZE     32
```

Comment:
Please revert this change and place it back.

Reply:
Done